doc/api/index.md

```diff
@@ -7,6 +7,7 @@
 
 * [About this documentation](documentation.md)
 * [Usage and example](synopsis.md)
+* <a href="index.html">Index</a>
```

Comment:
```suggestion
* [Index](index.md)
```

Reply:
I did this at the beginning, but then the linter complained. So I used the alternative version.

Reply:
This links the file to itself. What was the motivation for adding this link? It seems to result in a not great experience. Let's say I'm at this page:

![image](https://user-images.githubusercontent.com/718899/151711205-5da7e04e-fe96-468b-8dc4-eb80b62ff17f.png)

If I click on the index link in the left nav, all it does is load the left nav into the main page area:

![image](https://user-images.githubusercontent.com/718899/151711246-b94fa024-cf2c-4ee6-85b9-4524eafdb971.png)

Other than making the "Edit on GitHub" link available on this page, I don't know what value that adds for the user. Worse, possibly, now there's an "Index" link in the page that you can click that does...nothing.